.gitignore

```diff
@@ -2,3 +2,4 @@ dist
 node_modules
 .nyc_output/
 coverage/
+npm-debug.log
```

Comment:
Added this because I was pretty sure that even though this is useful for helping people troubleshoot, it probably isn't something you'd really want in the history.